json_validation/repositories/config_repository.py

```
@@ -0,0 +1,232 @@
+        Args:
+            config_file: Path to config file
+
+        Returns:
+            List of search values
+
+        Raises:
+            ConfigurationError: If file cannot be read
+        """
+        path = Path(config_file)
+        if not path.exists():
+            raise ConfigurationError(f"Config file not found: {config_file}")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)